### PR TITLE
update grpc readme information about protoc and streams

### PR DESCRIPTION
**Pull request by dsudia**
@peterbourgon 

As per our conversation in Slack, a link to the protocol releases page and some information about streams and go-kit. Let me know if any wording should be changed. Thanks!

### CODE REVIEW

File: transport/grpc/README.md [hunk 1]
```diff
@@ -2,18 +2,23 @@
 
 [gRPC](http://www.grpc.io/) is an excellent, modern IDL and transport for microservices.
 If you're starting a greenfield project, Go kit strongly recommends gRPC as your default transport.
-And using gRPC and Go kit together is very simple.
+
+One important note is that while gRPC supports streaming requests and replies, go-kit does not. You can still use streams in your service, but their implementation will not be able to take advantage of many go-kit features like middleware.
```
Comment (peterbourgon) [inline transport/grpc/README.md]: Please wrap at 80 columns. Also, "Go kit", not "go-kit".

**Comment by peterbourgon** [comment]
Ping? :)

**Comment by dsudia** [comment]
sorry, I apparently was not receiving notifications for conversations and PRs. that's fixed and now I'm gonna fix this

**Comment by dsudia** [comment]
@peterbourgon style changes up

**Comment by dsudia** [comment]
let me know if you want me to squash these

**Comment by peterbourgon** [comment]
Thanks!

**Pull request merged**